# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/jimmy/nullmailer/no-deprecated.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./lib/list.h.orig	2002-12-16 22:24:27.000000000 +0200
+++ ./lib/list.h	2004-06-12 13:28:00.069138384 +0300
@@ -111,8 +111,8 @@
     }
 private:
   const list<T>& lst;
-  const list<T>::node* prev;
-  const list<T>::node* curr;
+  const typename list<T>::node* prev;
+  const typename list<T>::node* curr;
 };
 
 template<class T>
@@ -164,8 +164,8 @@
     }
 private:
   list<T>& lst;
-  list<T>::node* prev;
-  list<T>::node* curr;
+  typename list<T>::node* prev;
+  typename list<T>::node* curr;
 };
 
 template<class T>
